issue using blob CLI method to submit with data `'"gm"'`

### Celestia Node version

v0.11.0-rc2

### OS

macOS

### Install tools

same as docs for arabica + light node

### Others

_No response_

### Steps to reproduce it

```bash
celestia rpc blob Submit 0x0c204d39600fddd3 '"gm"'
```

### Expected result

```bash
{
  "jsonrpc": "2.0",
  "result": HEIGHT_OF_PFB,
  "id": 1
}
```

### Actual result

panic: runtime error: index out of range [0] with length 0

### Relevant log output

```Shell
celestia rpc blob Submit 0x0c204d39600fddd3 '"gm"'
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
encoding/base64.(*Encoding).Encode(0x16f40b861?, {0x0?, 0x0?, 0x0?}, {0x140017f1c20?, 0x0?, 0x0?})
	/usr/local/go/src/encoding/base64/base64.go:155 +0x288
main.parseParams({0x16f40b85a, 0x6}, {0x1400013b6a0, 0x2, 0x0?})
	/Users/joshstein/celestia-node/cmd/celestia/rpc.go:141 +0x454
main.glob..func5(0x104de4320?, {0x1400013b680?, 0x4?, 0x4?})
	/Users/joshstein/celestia-node/cmd/celestia/rpc.go:100 +0x70
github.com/spf13/cobra.(*Command).execute(0x104de4320, {0x1400013b600, 0x4, 0x4})
	/Users/joshstein/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x5ac
github.com/spf13/cobra.(*Command).ExecuteC(0x104de4040)
	/Users/joshstein/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x340
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/joshstein/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/Users/joshstein/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:961
main.run()
	/Users/joshstein/celestia-node/cmd/celestia/main.go:28 +0x58
main.main()
	/Users/joshstein/celestia-node/cmd/celestia/main.go:21 +0x1c
```


### Notes

this worked before, with the format `'"text in plain text"'`

--- a/cmd/celestia/rpc.go
+++ b/cmd/celestia/rpc.go
@@ -138,6 +138,8 @@ func parseParams(method string, params []string) []interface{} {
 			blobData = decoded
 		case strings.HasPrefix(params[1], "\""):
 			// user input an utf string that needs to be encoded to base64
+			src := []byte(params[1])
+			blobData = make([]byte, base64.StdEncoding.EncodedLen(len(src)))
 			base64.StdEncoding.Encode(blobData, []byte(params[1]))
 		default:
 			// otherwise, we assume the user has already encoded their input to base64
@@ -178,6 +180,8 @@ func parseParams(method string, params []string) []interface{} {
 			blobData = decoded
 		case strings.HasPrefix(params[3], "\""):
 			// user input an utf string that needs to be encoded to base64
+			src := []byte(params[1])
+			blobData = make([]byte, base64.StdEncoding.EncodedLen(len(src)))
 			base64.StdEncoding.Encode(blobData, []byte(params[3]))
 		default:
 			// otherwise, we assume the user has already encoded their input to base64

